Functions.Templates/Templates/RabbitMQTrigger-CSharp/RabbitMQTriggerCSharp.csx

```diff
@@ -0,0 +1,6 @@
+using System;
+
```

Comment:
Do .csx and .cs templates belong to the same directory?

Reply:
yes, just like the rest of the samples.